cli/command/image/build.go

```diff
@@ -102,10 +103,6 @@ func NewBuildCommand(dockerCli command.Cli) *cobra.Command {
 			options.context = args[0]
 			return runBuild(cmd.Context(), dockerCli, options)
 		},
-		Annotations: map[string]string{
-			"category-top": "4",
-			"aliases":      "docker image build, docker build, docker buildx build, docker builder build",
```

Comment:
Why remove the aliases?
IIUC, this will be hit only with `DOCKER_BUILDKIT=0`, so I see why we'd want to remove `docker buildx build`, but not sure what's the reason behind removing the other aliases?

Reply:
Good point... I can add back the other aliases. They still apply with `DOCKER_BUILDKIT=0`